src/settings.test.js

```diff
@@ -0,0 +1,134 @@
+import { expect } from 'chai';
+import _ from 'lodash';
+import settings from './settings';
+import utils from './utils';
+
+
+describe('/settings', () => {
+  describe('.set()', () => {
+    describe('path is a string', () => {
+      it('sets chosen parameters with current values', () => {
+        const testSettings = _.cloneDeep(settings);
+        testSettings.set('modes.VD.kSigma', 3);
+        expect(testSettings.now.modes.VD.kSigma).to.be.equal(3);
+      });
+    });
+
+    describe('path is not a string (is an object)', () => {
+      it('makes diff with settings.now and sets diff', () => {
+        const testSettings = _.cloneDeep(settings);
+
+        const testDiffs = _.cloneDeep(settings);
+        testDiffs.set('modes.VD.kSigma', 3);
+        testSettings.set(testDiffs.now);
+        expect(testSettings.now.modes.VD.kSigma).to.be.equal(3);
+      });
+    });
+  });
+
+  describe('.get()', () => {
+    it('gets current parameter', () => {
+      expect(settings.get('modes.VD.kSigma')).to.be.equal(1);
```

Comment:
Can we expect a default value to be equal to 1. The default number may change, and the test fails. Should we set kSigma=1 before getting it?

Reply:
Yes, that will be right, thank you! Edited.

Reply:
Done.